### PR TITLE
Fix #143: Highfi TopicRevision Updated

### LINKED ISSUE #143: Final UI: Topic review tab fragment [Blocked: #15, #142]
This is tracking implementing the high fidelity UI for the topic review tab.

https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/1adacef0-f245-4115-a0e0-d1c2e7d5a0bb/TP-Revision-/

https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/dad2a9ff-ce33-4f00-894a-ec07dbb12797/TP-Revision-

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes: #143 

This PR deals with the highfi changes in Topic Revision.

Mock: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/1adacef0-f245-4115-a0e0-d1c2e7d5a0bb/TP-Revision-/
https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/dad2a9ff-ce33-4f00-894a-ec07dbb12797/TP-Revision-

## Screenshots:
Screenshot_1586241120
Screenshot_1586241123

## Accessibility Test
This PR now passes all the accessibility tests. (NOTE: Earlier they were failing because of color code.)
as_21
as_22

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@mschanteltc The accessibility tests are failing because of color contrast. Can we change this color?

**Comment by mschanteltc** [comment]
> @mschanteltc The accessibility tests are failing because of color contrast. Can we change this color?

Use #BE563C for the orange color. I believe the accessibility requirements should pass with this.

**Comment by rt4914** [comment]
## Updated Screenshots:
Screenshot_1585897424
Screenshot_1585897428


## All accessibility tests passed for the fragment.
as_1

**Comment by rt4914** [comment]
> > @mschanteltc The accessibility tests are failing because of color contrast. Can we change this color?
> 
> Use #BE563C for the orange color. I believe the accessibility requirements should pass with this.

It worked. Thanks.

File: app/src/main/res/layout-land/topic_revision_summary_view.xml [hunk 1]
```diff
@@ -20,9 +20,7 @@
     app:cardBackgroundColor="@color/white"
     app:cardCornerRadius="8dp"
```
Comment (nikitamarysolomanpvt) [inline app/src/main/res/layout-land/topic_revision_summary_view.xml]: As per mock radius is 4dp

File: app/src/main/res/layout/topic_revision_summary_view.xml [hunk 2]
```diff
@@ -20,9 +20,7 @@
     app:cardBackgroundColor="@color/white"
     app:cardCornerRadius="8dp"
```
Comment (nikitamarysolomanpvt) [inline app/src/main/res/layout/topic_revision_summary_view.xml]: ditto

File: app/src/main/res/layout/topic_revision_summary_view.xml [hunk 3]
```diff
@@ -40,19 +39,11 @@
...
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/subtopic_image_view" />
-
       <TextView
         android:id="@+id/subtopic_title"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:background="@color/oppiaBrownDark"
         android:ellipsize="end"
         android:fontFamily="sans-serif-medium"
```
Comment (nikitamarysolomanpvt) [inline app/src/main/res/layout/topic_revision_summary_view.xml]: Please update as per mock

**Review by nikitamarysolomanpvt — CHANGES_REQUESTED** [review]
PTAL on the suggestions

File: app/src/main/res/layout/topic_revision_summary_view.xml (hunk 3 above)
Comment (rt4914) [reply app/src/main/res/layout/topic_revision_summary_view.xml]: Updated as per new mock.

File: app/src/main/res/layout/topic_revision_summary_view.xml (hunk 2 above)
Comment (rt4914) [reply app/src/main/res/layout/topic_revision_summary_view.xml]: Updated to `4dp`

File: app/src/main/res/layout-land/topic_revision_summary_view.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout-land/topic_revision_summary_view.xml]: Updated to `4dp`

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

**Pull request merged**